Formula/kong.rb

```diff
@@ -92,3 +94,53 @@ def install
  
  role = traditional
  kic = off
+diff -r -u a/kong/cmd/prepare.lua b/kong/cmd/prepare.lua
+--- a/kong/cmd/prepare.lua	2022-09-12 14:38:55.000000000 +0200
++++ b/kong/cmd/prepare.lua	2022-09-15 10:53:58.000000000 +0200
+@@ -23,8 +23,8 @@
+
+ Example usage:
+  kong migrations up
+- kong prepare -p /usr/local/kong -c kong.conf
+- nginx -p /usr/local/kong -c /usr/local/kong/nginx.conf
++ kong prepare -p HOMEBREW_PREFIX -c kong.conf
```

Comment:
I am not fully understanding how HOMEBREW_PREFIX is working on this diff. If this is an environment variable, shouldn't it be preceded by `$` ?

```suggestion
+ kong prepare -p $HOMEBREW_PREFIX -c kong.conf
```

Perhaps homebrew is doing some magic on the diffs that I am missing.

Reply:
Found it in [the docs](https://docs.brew.sh/Formula-Cookbook#patches):

> In embedded patches, the string “HOMEBREW_PREFIX” is replaced with the value of the constant HOMEBREW_PREFIX before the patch is applied.